.gitignore

```diff
@@ -56,6 +56,8 @@ src/qt/bitcoin-qt.creator
 src/qt/bitcoin-qt.creator.user
 src/qt/bitcoin-qt.files
 src/qt/bitcoin-qt.includes
+# An intermediate XLIFF translation file.
+src/qt/locale/bitcoin_en.xlf
```

Comment:
Transifex auto-updates from the branch. If you want transifex to pick this file up instead of the `.ts` it needs to be checked in, not added to `.gitignore`.

Reply:
Thanks! Updated.